zubhub_frontend/zubhub/src/components/materialsUsed/materialsUsed.jsx

```diff
@@ -0,0 +1,127 @@
+import React, {useState} from 'react';
+import AddMore from '../../components/addMore/addMore';
+import clsx from 'clsx';
+import ImageIcon from '@material-ui/icons/Image';
+import AddIcon from '@material-ui/icons/Add';
+import CustomButton from '../../components/button/Button';
+import Input from '../input/input'
+import {
+  Grid,
+  Box,
+  Container,
+  Card,
+  CardActionArea,
+  CardContent,
+  Chip,
+  Dialog,
+  Select,
+  MenuItem,
+  Typography,
+  CircularProgress,
+  OutlinedInput,
+  FormHelperText,
+  FormControl,
+  InputLabel,
+  ClickAwayListener,
+} from '@material-ui/core';
+import {handleImageFieldChange} from '../../views/create_activity/createActivityScripts';
+import UploadFile from '../../components/upload_file/uploadFile';
+
+function MaterialsUsed(props) {
+  const refs = {
+    imageInput: React.useRef(null),
+    image_count_el: React.useRef(null),
+  }
+  const {
+        classes, 
+        common_classes, 
+        addMoreLabel,
+        imagesLabel, 
+        encouragingText,
+        setValue,
+        materialsUsedList,
+        setNewActivity
+        } = props;
+  const [materials, setMaterials] = useState(materialsUsedList)     
+  const handleImageButtonClick = (e) => {
+    refs.imageInput.current.click()
+  } 
+  
+  return (
+    <div>
+      <Grid container>
+        <Grid item xs={12} md={6}>
+          {materials.map((value, index) => (
+            <Input
+              label={`materialsUsed[${index}]`}
+              defaultValue={
+                props.newActivity['materialsUsed']
+                  ? props.newActivity.materialsUsed[index]
+                  : ''
+              }
+              classes={classes}
+              {...props}
+            />
+          ))}
+        </Grid>
+        <Grid
+          item
+          xs={12}
+          md={6}
+          spacing={3}
+          container
+          direction="row"
+          alignItems="flex-end"
+          justifyContent="center"
+        >
+          <AddMore setNodeList={setMaterials} label={addMoreLabel} />
+        </Grid>
+      </Grid>
+      <p className={clsx(classes.fieldHelperTextStyle, classes.errorMessage)}>
+        {props.touched['materialsUsed'] &&
+          props.errors['materialsUsed'] === 'required1' &&
```

Comment:
is the error supposed to be required or required1?

Reply:
i thought that i might need two required messages but this was stupid 😅, i will change it